tests/core/test_config.py

```diff
@@ -223,51 +222,35 @@ def check_memoization(p):
         assert p.max_history == 5
         assert p.priority == 3
 
-    def check_fallback(p):
-        assert p.fallback_action_name == "action_default_fallback"
-        assert p.nlu_threshold == 0.7
-        assert p.core_threshold == 0.7
-        assert p.priority == 2
-
-    def check_form(p):
+    def check_rule(p):
         assert p.priority == 1
 
     ensemble_dict = {
         "policies": [
             {"max_history": 5, "priority": 3, "name": "MemoizationPolicy"},
```

Comment:
Might it make sense to use the new default for `max_history` here?

Reply:
Adding a test for that.